cmd/webhook/main.go

```diff
@@ -14,7 +14,41 @@
 
 package main
 
-func main() {
+import (
+	"flag"
+
+	"antrea.io/resource-auditing/pkg/webhook"
+	"antrea.io/resource-auditing/pkg/webhook/gitmanager"
+	"k8s.io/klog/v2"
+)
+
+func processArgs(portFlag *string, dirFlag *string) {
+	flag.StringVar(portFlag, "p", "8080", "specifies port that audit webhook listens on")
+	flag.StringVar(dirFlag, "d", "", "specifies directory where resource repository is created")
```

Comment:
maybe default this to `"/data/resource-auditing-repo"` rather than `""`

Reply:
backend logic interprets empty string as defaulting to current working directory - should I change the default to "/data/resource-auditing-repo" and not use current working directory at all then?

Reply:
I think the current behavior is fine then, but maybe add that information to the usage string